src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,13 +1,15 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar" style="display: none"></div>
 {% include '@WebProfiler/Profiler/base_js.html.twig' %}
 <script>/*<![CDATA[*/
-    (function () {
+    document.addEventListener('DOMContentLoaded', function () {
```

Comment:
@javiereguiluz the toolbar itself is injected in the page being displayed, so it is good to avoid breaking the page in older browsers (as it would make it impossible to test your site properly in older browsers).
This is why we applied bug fixes regarding the toolbar in IE8 in the past for instance (while not supporting IE8 in the profiler pages, as these can be viewed in a modern browser even when testing the page itself in an old browser)